src/helper/Helper.tsx

```diff
@@ -0,0 +1,20 @@
+import { CompletedFilter } from '../types/CompletedFilter';
+import { Todo } from '../types/Todo';
+
+export const filteredTodos = (
+  todos: Todo[],
+  completedFilter: CompletedFilter,
+) => {
+  return todos.filter(todo => {
```

Comment:
It would be great, if you don't filter the array if `completedFilter` equals `all`

Reply:
'all' is missing in my switch case or you mean - I need to write something another in default?

Reply:
Don't filter array if `completedFilter` equals `All`

Reply:
```
switch (status) {
  case "Completed"
   return todos.filter(todo => todo.completed)
  case "All":
  default: 
   return todos;
}